crypto/ui/ui_util.c

```diff
@@ -109,7 +104,8 @@ static int ui_read(UI *ui, UI_STRING *uis)
             char result[PEM_BUFSIZE];
             const struct pem_password_cb_data *data =
                 UI_method_get_ex_data(UI_get_method(ui),
-                                      ui_method_data_index());
+                                      RUN_ONCE(&get_index_once,
+                                               ui_method_data_index));
```

Comment:
> the other places with index do it that way as i recall.

Yes. They often create objects that can't be returned as int or initialize stuff that don't have a sensible value to return...